src/foundry/client/apps/forms/ownership.d.ts

```diff
@@ -31,7 +31,7 @@ declare global {
   /**
    * @deprecated since v10.
    */
-  class PermissionControl extends DocumentOwnershipConfig {}
+  class PermissionControl extends DocumentOwnershipConfig<any, any> {}
```

Comment:
`DocumentOwnershipConfig` takes generic parameters now, since `PermissionControl` is deprecated I figured typing anything specific didn't matter.